apps/publish/enqueue/enqueue_service.py

```diff
@@ -254,11 +254,17 @@ def publish(self, doc, target_media_type=None, content_type=None):
         :raises PublishQueueError.item_not_queued_error:
                 If the nothing is queued.
         """
+        is_publish = True
```

Comment:
imo the name is a bit misleading, it's always publishing something, should be probably `first_version` or something